argocd/minio/values.yaml

```diff
@@ -0,0 +1,12 @@
+minio:
+  accessKey: ZQQhDzrKcSB2eYntvQEcfE5m8xgBwdMd
```

Comment:
Can you please try to create a secret in Vault using Terraform and use the `existingSecret` field instead? There is an example in the demo-app